Makefile

```diff
@@ -0,0 +1,151 @@
+# Makefile for Salesforce CDP Connector packages
+# Supports both salesforcecdpconnector and salesforcecdpconnector-grpc packages
+
+.PHONY: help clean build build-main build-grpc install install-main install-grpc install-dev install-dev-main install-dev-grpc uninstall uninstall-main uninstall-grpc test test-main test-grpc lint lint-main lint-grpc all
```

Comment:
All of these should be updated to use uv, and probably use workspaces: https://docs.astral.sh/uv/concepts/projects/workspaces/#when-not-to-use-workspaces